app/components/auk/progress-bar.js

```diff
@@ -1,7 +1,7 @@
 import Component from '@glimmer/component';
 
 /**
- * @argument progress <Number> in percent (0-100)
+ * @argument {Number} progress in percent (0-100)
```

Comment:
Er mis hier precies dat je `@progress` moet gebruiken
Stel voor om dit aan te passen naar

`* @argument progress {Number}: progress in percent (0-100)`
of
`* @argument {Number} progress: progress in percent (0-100)`

Beide stijlen zijn gebruikt in de auk klassen, dus weet niet wat de meest gangbare is